Inc/software_info.h

```diff
@@ -32,13 +32,16 @@
  *
  ****************************************************************************/
 
-#ifndef INC_VERSION_H_
-#define INC_VERSION_H_
+#ifndef INC_SOFTWARE_INFO_H_
+#define INC_SOFTWARE_INFO_H_
 
 #include <stdint.h>
 #include <stdbool.h>
 
-bool Version_getDataJson(uint8_t* buffer, uint16_t size);
-bool Version_getData(uint8_t* buffer, uint16_t size);
+#define GET_VERSION_CMD      "version"               //!< String command for bootloader to send version
+#define GET_SW_INFO_JSON_CMD "software_info_json"    //!< String command for bootloader to send software info in JSON format

```

Comment:
In my opinion, it should be removed from here (from utility).
We can define these commands in bootloader where we need them and where they will be used.

Reply:
Ok, I'll remove it from now.  I added it since the bootloader and firmware are sharing the same command for this purpose, but I can understand why this can seems an inappropriate place to keep it. 
We should consider if we want the utility to be a general-purpose utility or utility for IMBottloder and IMLedBlink repositories.